package-lock.json

```diff
@@ -10052,7 +10050,7 @@
       "dev": true
     },
     "parse-server": {
-      "version": "github:parse-community/parse-server#ac40bf66f7cdeeb578fd716a8e3dd540470209ca",
+      "version": "github:parse-community/parse-server#bde96a9002a3286c417efaae79856264b8788cb1",
```

Comment:
Can you revert this package-lock.json? It points to an old commit and your tests will fail.

https://github.com/parse-community/parse-server/commit/bde96a9002a3286c417efaae79856264b8788cb1

Reply:
Done, I wonder why it always points to the old commit after I run npm install.

Reply:
You have to delete your node module folder

Reply:
Because package.json says `"parse-server": "github:parse-community/parse-server#master"`

Reply:
That was it!